examples/searchR1/data/searchr1_dataset.py

```
@@ -0,0 +1,362 @@
+            try:
+                shutil.copy(from_path, to_path)
+                returncode = 0
+            except shutil.SameFileError:
+                returncode = 0
+            except Exception as e:
+                logger.warning(f"copy {from_path} {to_path} failed: {e}")
+                returncode = -1
+    return returncode == 0
+
+
+def _run_cmd(cmd: str, timeout=None):
```

Ruff (0.12.2)

Unused function argument: `timeout`

(ARG001)